### PR TITLE
Scaffold out figma

### PR DESCRIPTION
**Pull request by nickadamson**

### CODE REVIEW

**Comment by nickadamson** [comment]
GH actions are not testing components, only e2e

**Comment by aplyd** [comment]
lol pain

**Comment by nickadamson** [comment]
you already know we are squash merging this. 
goodnight.

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Cypress Tests
+
+on: [push]
```
Comment (aplyd) [inline .github/workflows/cypress.yml]: are we sure we want to run this workflow every push instead of pull-request?

**Review by aplyd — APPROVED** [review]
Solid work on this. Looks good to me

**Comment by aplyd** [comment]
also apologies on the delay

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (nickadamson) [reply .github/workflows/cypress.yml]: no, pull-request would be more apt. :pray:

Comment (nickadamson) [reply .github/workflows/cypress.yml]: fix - https://github.com/offtheair/onthechain/pull/3/commits/802e4613d4e68c36220041fc271b2216a9aab858

**Comment by aplyd** [comment]
let's gooooo

**Pull request merged**